gym_xiangqi/xiangqi_game.py

```diff
@@ -133,6 +171,47 @@ def load_piece_images(self, pieces: list):
             pieces[i].set_basic_image()
             pieces[i].set_select_image()
 
+    def to_real_coor(self, clicked_coor):
+        clicked_real_x = (clicked_coor[0] - COOR_OFFSET) // COOR_DELTA
+        clicked_real_y = (clicked_coor[1] - COOR_OFFSET) // COOR_DELTA
+
+        return clicked_real_x, clicked_real_y
+
+    def find_target_piece(self, clicked_coor):
+
+        clicked_x = clicked_coor[0]
+        clicked_y = clicked_coor[1]
+
+        for piece in self.agent_piece[1:]:
+
+            piece_x = piece.get_pygame_coor()[0]
+            piece_y = piece.get_pygame_coor()[1]
```

Comment:
How about this `piece_x, piece_y = piece.get_pygame_coor()`

Reply:
on_event, to_real_coor, find_target_piece

Reply:
fixed

Reply:
> How about this `piece_x, piece_y = piece.get_pygame_coor()`

fixed! However, sometimes the code gets separated due to the "Line too long" error message from flack8Lint. We might need to discuss somehow ignoring this error if we can.

Reply:
Yea the standard line length is 80, and you can always use bracket to get new line. Something like this:

```
piece_x, piece_y = (
    piece.get_pygame_coor())
```